pkg/client/client.go

```diff
@@ -199,4 +214,38 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse) {
 			}
 		}
 	}
+
+	if sr.Controllers != nil {
+		nFailing, out := 0, []string{"  Name\tLast success\tLast error\tCount\tMessage\n"}
+		for _, ctrl := range sr.Controllers {
+			if status := ctrl.Status; status != nil {
```

Comment:
Since we only do something when `status != nil`, we can continue when `status == nil` and avoid the indent.

Reply:
fixed